src/pages/Manage/TablePage.tsx

```diff
@@ -66,14 +80,29 @@ export default function TablePage() {
         </div>
       ) : (
         <>
-          <div className="flex h-full flex-wrap content-start gap-5 overflow-y-scroll p-10">
-            {tableList.map(({ tableNo }) => (
-              <QRCard tableNo={tableNo} key={tableNo}/>
+          <div
+            className="flex h-full flex-wrap content-start gap-5 overflow-y-scroll p-10"
+            id="qrDiv"
+          >
+            {tableList.map(({ tableNo, url }) => (
+              <QRCard tableNo={tableNo} key={tableNo} qrUrl={url} />
             ))}
           </div>
           <div className="flex justify-center gap-2 py-[6px]">
-            <Button title="전체 QR 이미지 저장" type="others" />
-            <Button title="전체 QR 프린트" type="others" />
+            <Button
+              title="전체 QR 이미지 저장"
+              type="others"
+              onClick={() => {
+                downloadAllQR();
+              }}
```

Comment:
요런식으로 간략하게 표현해주시면 더 좋습니다~
```suggestion
              onClick={downloadAllQR}
```